integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -450,5 +450,61 @@ describe('Native ETH value integration tests', () => {
         expect(returndata).to.eq('0x')
       })
     })
+
```

Comment:
I would recommend moving this test block into `native-eth.spec.ts` (sorry for the near-collision with this one's name).  This testfile is pretty hefty and exclusively deals with testing the behavior of `ovmCALL` with nonzero value.  The other one is where we have other tests for `OVM_ETH` like depositing, transferring, and withdrawing.